modules/cert-manager-istio-csr-updating.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-integrating-istio.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cert-manager-istio-csr-updating_{context}"]
+= Updating the {cert-manager-operator} with istio-csr 
+
+[IMPORTANT]
+====
+When the Istio-CSR TechPreview feature gate is enabled, the operator cannot be upgraded. To upgrade to the next available version, you must uninstall the Operator and remove all {cert-manager-operator} and Istio-CSR resources before reinstalling it.
```

Comment:
Also, the text itself needs to replace "upgrade" with update, and Operator needs to be capitalized. Lastly, I think the second use of "update" can be replaced with "use", since technically you're not performing an update by deleting and reinstalling with a new version:
```suggestion
When the Istio-CSR TechPreview feature gate is enabled, the Operator cannot be updated. To use to the next available version, you must uninstall the Operator and remove all {cert-manager-operator} and Istio-CSR resources before reinstalling it.
```

Reply:
`Upgrade` is more accurate here.